vignettes/wwinference.Rmd

```diff
@@ -353,7 +367,21 @@ state-level R(t) estimate.
 We can generate this directly on the output of `wwinference()` using:
 ```{r extracting-draws}
 draws_df <- get_draws_df(ww_fit)
+
+# Print the names of the variables included in the output dataframe
```

Comment:
I don't think this comment is needed.
```suggestion
```

Reply:
See comment below

Reply:
To clarify, I think it's fine to print the names of the variables, I just don't think we need a comment to say we're doing it.